### ISSUE TITLE
(Accessibility)A11y_VsCode_ShowOpenedEditors_Keyboard:Keyboard focus does not come  to the"Group Right" .

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51873]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
Keyboard,MAS2.1.1
**Repro Steps:**
1)Launch VSCode.
2Navigate   to welcome screen and beside opened editors pane at top(in middle   pane)->Select More(…)->Select "Show Opened Editors" from   dropdown.
3)Check with NVDA reading.
**Actual:**
Keyboard focus does not come  to the"Group Right"
NVDA also does not read the "Group Right".
**.Expected:**
NVDA should read the placeholder which is present on the screen.
NVDA should  read the "Group Right".
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
**[User] Impact:**
User will not able to know that there is placeholder present on the screen.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
**Attachment for Reference:**
[(Accessibility)A11y_VsCode_ShowOpenedEditors_Keyboardtab.pptx](https://github.com/Microsoft/vscode/files/2101642/Accessibility.A11y_VsCode_ShowOpenedEditors_Keyboardtab.pptx)
![nvda_speechviewer](https://user-images.githubusercontent.com/39204909/41773296-3357550e-7639-11e8-9145-ba48a191d89e.png)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51850
